src/app/pages/projectmanagement/projectmanagement.component.ts

```diff
@@ -80,7 +81,8 @@ export class ProjectmanagementComponent implements OnInit {
 
   getListProject() {
     this.projectService.getAllProject().subscribe((response: ResponseProjectDTO) => {
-      this.projects = response.data;
+      this.projectsClone = response.data;
+      this.projects = Object.assign(this.projectsClone)
```

Comment:
this.projectsClone = response.data;
      this.projects = Object.assign(this.projectsClone)

có thể bị sai tên hay không ?

Reply:
ok